pom.xml

```diff
@@ -48,7 +48,7 @@
 		<maven.compiler.release>8</maven.compiler.release>
 		<scala.minor.version>2.11</scala.minor.version>
 		<scala.version>${scala.minor.version}.11</scala.version>
-		<smartdatalake.version>1.0.2</smartdatalake.version>
+		<smartdatalake.version>1.0.4-SNAPSHOT</smartdatalake.version>
```

Comment:
This won't work. I configured a minimal Action that builds the package, therefore the referenced SDL version needs to be in maven central. 
But we can discuss that: the action pipeline is not mandatory on SDL-examples to be honest.

Reply:
You're right, lets go to the release version